src/resources/postcss/views/month/_multiday-event.pcss

```diff
@@ -42,4 +75,42 @@
 	.tribe-events-calendar-month__event-multiday-title {
 		font-weight: var(--font-weight-regular);
 	}
+
+	.tribe-events-calendar-month__event-multiday-width-1 {
+		width: 100%;
+	}
+
+	.tribe-events-calendar-month__event-multiday-width-2 {
+		width: calc( 200% + 2px );
+	}
+
+	.tribe-events-calendar-month__event-multiday-width-3 {
+		width: calc( 300% + 3px );
+	}
+
+	.tribe-events-calendar-month__event-multiday-width-4 {
+		width: calc( 400% + 4px );
+	}
+
+	.tribe-events-calendar-month__event-multiday-width-5 {
+		width: calc( 500% + 5px );
+	}
+
+	.tribe-events-calendar-month__event-multiday-width-6 {
+		width: calc( 600% + 6px );
+	}
+
+	.tribe-events-calendar-month__event-multiday-width-7 {
+		width: calc( 700% + 7px );
+	}
+
+	.tribe-events-calendar-month__event-multiday--empty,
+	.tribe-events-calendar-month__event-multiday--hidden {
+		display: none;
+
+		@media (--viewport-medium) {
+			visibility: hidden;
```

Comment:
This style is fine for `--empty`, but `--hidden` should use an `opacity: 0` style.

Reply:
`--empty` can have an `opacity:0;` but for hidden need to be have `visibility: hidden;` because if not, the element comes next in the dom and the hover for multiday doesn't work.

Reply:
not sure i follow you here. isn't `--empty` supposed to be a blank spacer? that should work with no styles for opacity or visibility, just height/padding and margin. but for `--hidden`, yes the point is for that to lay above the visible multiday. That is what the JS handles to allow for those on hover and focus to display the correct styles on the visible multiday event. That is what's happening below:

https://www.loom.com/share/7146f385e65f4911b09aa6b052815476

Reply:
Ok, didn't know that you were taking that for the JS, thanks for the clarification.